admin/code/LeftAndMain.php

```diff
@@ -536,11 +536,10 @@ protected function init() {
 			window.ss.config = " . $this->getCombinedClientConfig() . ";
 		");
 
-		Requirements::javascript(ltrim(FRAMEWORK_ADMIN_DIR . '/client/dist/js/bundle-lib.js', '/'));
+		Requirements::javascript(FRAMEWORK_ADMIN_DIR . '/client/dist/js/vendor.js');
```

Comment:
Can we use ltrim() to be consistent? @sminnee designed this to work so that we can run tests from the base dir (BASE_DIR === MODULE_DIR).